cypress.config.js

```diff
@@ -0,0 +1,36 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  video: true,
+  chromeWebSecurity: true,
+  fixturesFolder: '.cypress/fixtures',
+  screenshotsFolder: '.cypress/screenshots',
+  videosFolder: '.cypress/videos',
+  downloadsFolder: '.cypress/downloads',
+  viewportWidth: 2000,
+  viewportHeight: 1320,
+  requestTimeout: 60000,
+  responseTimeout: 60000,
+  defaultCommandTimeout: 60000,
+  //experimentalNetworkStubbing: true,
```

Comment:
Do we need these settings, we can delete if we don't need

Reply:
They are the recommended fixes to a chrome crashing error I have thoroughly tested https://docs.cypress.io/guides/references/error-messages#The-Chromium-Renderer-process-just-crashed.
Leaving them in to indicate that it has been tried but was not effective. If we end up bumping the version in the future these might be used but lowering the test kept in memory has been the only effective fix. I think keeping them in the file as settings that can be turned on might help in the future.